topk-js/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+edition = "2021"
+name = "topk-js"
+description = "TopK JavaScript SDK with TypeScript support"
+version = "0.0.0"
+repository = "https://github.com/fafolabs/topk-sdk"
+homepage = "https://topk.io"
+documentation = "https://docs.topk.io"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+# Default enable napi4 feature, see https://nodejs.org/api/n-api.html#node-api-version-matrix
+napi = { version = "3.0.0-alpha.33", default-features = false, features = ["napi4", "tokio_rt", "serde-json"] }
```

Comment:
nit: let's add a TODO to upgrade to a stable version when it's released. @petertoth any idea when that's going to happen?

Reply:
one thing to watch out for I guess -- [this comment](https://github.com/napi-rs/napi-rs/issues/1493#issuecomment-2746275863) mentions windows compatibility in relation to napi 3 release, worth verifying